editor/filesystem_dock.cpp

```diff
@@ -494,43 +494,22 @@ void FileSystemDock::_update_display_mode(bool p_force) {
 
 void FileSystemDock::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_TRANSLATION_CHANGED:
-		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED:
 		case NOTIFICATION_ENTER_TREE: {
 			if (initialized) {
 				return;
 			}
 			initialized = true;
```

Comment:
Since this is used purely for one-time initialization, it could just use `READY`, no?

Reply:
This is probably part of the original code there that was just refactored? I think Its beyond the scope of the PR